geps/gep-1709/index.md

```diff
@@ -231,6 +231,22 @@ move on to [certification](#certification) to report the results.
 [go]:https://go.dev
 [lib]:https://pkg.go.dev/sigs.k8s.io/gateway-api@v0.6.2/conformance/utils/suite
 
+### Implementation mode
+
+The certification process runs against an implementation using a specific mode.
+The final report will contain such a mode in the modes field. A "mode" is intended
+to capture situations where a Gateway API implementation may have different features
+and capabilities depending on how it is deployed (e.g. an implementation might be
+deployed with an external, OR internal load balancer, and have different capabilities
+depending on the mode chosen). The modes are implementation-specific, and no upstream
+mode is defined, except for `plain`, the default one used in case no mode is specified.
```

Comment:
Why not just call the default mode "default"? It seems at least theoretically plausible that "plain" would have a real meaning for some implementations that was also non-default.

Reply:
yep, I've replaced `plain` with `default`